framework/global/realfn.h

```diff
@@ -0,0 +1,111 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_FRAMEWORK_REALFN_H
```

Comment:
I didn't understand the purpose of this file, we have qFuzzyCompare(float/double) and qFuzzyIsNull(float/double)
It's like a reinventing the wheel

Reply:
This file was added for the following reasons:
* Where it’s easy not to use Qt, it’s better not to use Qt (for example, in an audio engine)
* This file has a number of additional convenient comparison functions <= or >=